storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,126 @@
+.. _page_tracking:
+
+====================================
+Page Tracking for Incremental Backup 
+====================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27 adds support for the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup with page tracking, Percona XtraBackup (PXB) uses the MySQL mysqlbackup component. This component looks in the InnoDB data files for pages that have been modified since the last backup and copies only those changed pages. This operation removes the need to scan all of the pages in the database. If the number of changed pages is not over 50%, the page tracking feature can improve the speed of incremental backups.
+
+Install the component
+---------------------
+
+To start using the page tracking functionality, do the following steps:
+
+1. Install the mysqlbackup component and enable it on the server: 
+
+  .. code-block:: bash
+
+     INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+.. note::
+
+   The mysqlbackup component processes all the tracked pages in memory. If the component does not have enough available memory to process all the tracked pages, the component throws an error and exits. For example, if an incremental backup uses 200GB, the component can additionally use about 100MB of memory to store the page tracking data.
+
+Using page tracking
+-------------------
+
+If the server page flushing rate is low or it uses a big buffer pool, you can start page tracking before creating the first backup. Thus, you ensure that page tracking log sequence number (LSN) is more than the checkpoint LSN of the server.
```

Comment:
In the usage section
first mention we can start page tracking on server during the full or incremental  backup with option --page-tracking  provide an example. 
and then give example of how to take incremental backup using page-tracking

The delay section we can add in note. Not so important for most of users.

Reply:
I provide the mentioned info in the Enable page tracking section.